src/js/components/Box/doc.js

```diff
@@ -29,6 +29,31 @@ const ANIMATION_SHAPE = PropTypes.shape({
   size: PropTypes.oneOf(['xsmall', 'small', 'medium', 'large', 'xlarge']),
 });
 
+const BACKGROUND_SHAPE = PropTypes.oneOfType([
+  PropTypes.string,
+  PropTypes.shape({
+    color: PropTypes.string,
+    dark: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
+    image: PropTypes.string,
+    position: PropTypes.string,
+    opacity: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.bool,
+      PropTypes.number,
+      PropTypes.oneOf(['weak', 'medium', 'strong']),
+    ]),
+    repeat: PropTypes.oneOfType([
+      PropTypes.oneOf(['no-repeat', 'repeat']),
+      PropTypes.string,
+    ]),
+    size: PropTypes.oneOfType([
+      PropTypes.oneOf(['cover', 'contain']),
+      PropTypes.string,
+    ]),
```

Comment:
Interestingly we are not exposing these options on Box `background` prop. 
Might be a separate PR, but should we add them? https://v2.grommet.io/box#background

Reply:
Hmm, that's a good observation. I think I'll check first into why they're not being pulled into the docs site. Seems like if they're defined here that they should be exposed. Agreed, it should be a separate PR regardless.

Reply:
I will adjust the docs site in a separate PR to expose these options.